.pylintrc

```diff
@@ -654,3 +654,4 @@ min-public-methods=1
 # "BaseException, Exception".
 overgeneral-exceptions=BaseException,
                        Exception
+                       
```

Comment:
No change occurred in this file other than the extra line getting included. Please fix this and the PR will be ready to approve.